src/Tools/MicroBuild/copy-symbols.ps1

```diff
@@ -0,0 +1,35 @@
+# Copy our symbols to the Symbols directory.  This simplifies the logic of our achiving 
+# and indexing steps.  
+Param(
+    [string]$binariesPath = $null
+)
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+try
+{
+    $symbolsPath = join-path $binariesPath "Symbols"
```

Comment:
That link documents the rough structure. But symstore.exe is really the only thing that knows about how any of that actually is. If you were going to try putting it directly in that format, I'd just look at what it takes to shell out to it.

Some background: the insane glob we have is essentially an input to VSTS's task that calls symstore.exe and creates the Symbols folder with the actual layout (identifiable by the magic 000admin folder inside of it.) Your change at least means we could get rid of the insane glob and point it to this folder. I'm OK with that, it just means we have more copies of more stuff. (and we need some more temporary folder name here to not conflict).

Mostly a question of your pain tolerance.